CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 __Fixes__
 - Use a seperate Keychain for each app bundleId. This only effects macOS apps as their Keychain is handled by the OS differently. For macOS app developers only, the user who logged in last to your app will have their Keychain upgraded to the patched version. Other users/apps will either need to login again or logout then login again ([#224](https://github.com/parse-community/Parse-Swift/pull/224)), thanks to [Corey Baker](https://github.com/cbaker6).
 
+__Improvements__
```

Comment:
This is in the wrong place. It should be above 1.9.8 as that version is already released